applications/teletype/src/teletype_templates.erl

```diff
@@ -175,11 +176,16 @@ render(TemplateId, Macros, DataJObj, 'true') ->
 templates_source(_TemplateId, 'undefined') ->
     lager:warning("no account id for template ~s, no template to process", [_TemplateId]),
     'undefined';
-templates_source(_TemplateId, ?KZ_CONFIG_DB) ->
-    ?KZ_CONFIG_DB;
```

Comment:
This change may have side affect.
Is possible to cherry-pick this to `4.2`? Or this may be used only on `4.3`?